packages/fxa-content-server/tests/functional/email_service.js

```diff
@@ -24,6 +24,7 @@ registerSuite('email_service', {
         .then(fillOutSignUp(email, PASSWORD))
         .then(getEmailHeaders(user, 0))
         .then(headers => {
+          console.log('PHIL!', headers['x-email-service'], headers);
```

Comment:
😆

Reply:
Yeah, debugging it in CI, I should close the PR while I do that sorry!